createNativeStackNavigator/README.md

```diff
@@ -310,3 +302,13 @@ const largeHeaderInset = statusBarInset + 96; // inset to use for a large header
 ```
 
 You can also see an example of using these values with a `ScrollView` here: https://snack.expo.io/@wolewicki/ios-header-height.
+
+### Disabling `react-native-screens`
```

Comment:
Please move it to the place where there was information about enabling them. Same for below file.

Reply:
Fixed in e6e16cc492349bb0433e3ef952d8afe16fcd6f59